react/Footer/FooterLink/FooterLink.less

```diff
@@ -15,15 +15,5 @@
 }
 
 .newBadge {
-  display: inline-block;
-  text-transform: uppercase;
-  font-size: 12px;
-  font-weight: 500;
-  line-height: 18px;
-  height: 18px;
-  color: @sk-white;
-  background-color: @sk-purple;
-  padding: 0 @row-height;
-  border-radius: 9px;
-  margin-left: 8px;
+  margin-left: @gutter-width / 4;
```

Comment:
This should not proportionate to the gutter, its more accurately a non-breaking space character.